data/json/items/containers.json

```
@@ -187,6 +187,23 @@
     "symbol": ")",
     "color": "black"
   },
+  {
+    "id": "bag_garbage_reinforced",
+    "type": "GENERIC",
+    "category": "container",
+    "name": { "str": "reinforced garbage bag" },
+    "looks_like": "bag_canvas",
+    "description": "Plastic bags placed within each other made to carry much heavier loads, could even carry a dead body inside, it dosent stop the smell though.",
```

```suggestion
    "description": "Plastic bags placed within each other made to carry much heavier loads.",
```
It would need to be a pretty tiny person to fit in one garbage bag, no?

---

this mite be a regional misunderstanding, garbage bags where im from are those big 1.5m long plastic black ones that fill all the space in those bins the garbage trucks pick up, im not american so i defer to whatever the standards there would be